README.md

```diff
@@ -7,41 +7,122 @@ A CNAB bundle for installing [riff](https://projectriff.io/)
 To install this cnab bundle, you can use any available cnab runtime. The following are installation instructions using [duffle](https://duffle.sh/).
 
 ### Steps
-1. Download the latest duffle release for your operating system from duffle's [release page](https://github.com/deislabs/duffle/releases),
- put it somewhere on your path, and make it executable using, for example, `chmod +x /usr/local/bin/duffle`.
-1. Start a kubernetes cluster as described in the riff [getting started](https://projectriff.io/docs/getting-started/) documentation,
-but stop once you've created a cluster and, if appropriate, given yourself cluster-admin permissions.
- If using `minikube`, please run the following command before starting `minikube`:
+The high level steps to install riff are:
+1. Start a kubernetes cluster
+1. Install a CNAB runtime. We will use [duffle](https://duffle.sh/).
+1. Create service account in order to grant additional permissions like creating CRDs over the default sercice account.
+1. Use the duffle k8s driver to install riff. This will run riff bundle's invocationImage in k8s cluster using the service account created in the previous step. We will also set a couple of environment variables required by the k8s driver.
+
+### Detailed Instructions
+Follow the steps below for your choice of kubernetes cluster
+#### Minikube
+1. [Install minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/)
+1. Start the minikube cluster
     ```
-    minikube config set embed-certs true
+    minikube start --memory=4096 --cpus=4 \
+    --kubernetes-version=v1.14.0 \
+    --vm-driver=hyperkit \
+    --bootstrapper=kubeadm \
+    --extra-config=apiserver.enable-admission-plugins="LimitRanger,NamespaceExists,NamespaceLifecycle,ResourceQuota,ServiceAccount,DefaultStorageClass,MutatingAdmissionWebhook"
     ```
-1. Check that your kubeconfig file is configured to talk to that cluster:
+1. Confirm that your kubectl context is pointing to the new cluster
```

Comment:
Do we still need this? It's been a very long time since I've seen Minikube not set itself as the default context.